.github/workflows/ci.yml

```diff
@@ -50,13 +50,7 @@ jobs:
 
       - name: Compile
         run: |
-          mvn clean compile -U -Dmaven.javadoc.skip=true | grep -v "Downloading\|Downloaded"
-
-      - name: Install JDK 8
-        uses: actions/setup-java@v3
-        with:
-          java-version: '8'
-          distribution: 'zulu'
```

Comment:
duplicate install jdk step, remove it

<img width="413" alt="image" src="https://user-images.githubusercontent.com/17706099/212109099-7cce7711-4c4b-49d9-b48e-21e8db130ebb.png">